Common/src/gRPC/Services/Agent.cs

```diff
@@ -474,6 +500,19 @@ await foreach (var request in requestStream.ReadAllAsync(cancellationToken)
                                                           .ConfigureAwait(false);
                                         },
                                         cancellationToken);
+              findDependenciesTask = Task.Run(async () =>
+                                              {
+                                                var taskFound = await taskTable_.FindTasksAsync(data => data.DataDependencies.Contains(request.Init.Key),
```

Comment:
I wanted to keep tasks in the queue, so I need to get the task Ids.
I don't think that this request is well optmized since we search in a list from TaskData objects.
It was simpler this way. We can do so in a second time. Adding the reverse dependency list in the Results would have made more modifications.